src/routes.tsx

```diff
@@ -170,6 +171,11 @@ export const reactPageRoutes: ReactPageRoute[] = [
     Component: Referral,
     exact: true,
   },
+  {
+    path: LOCALE_PATH_PATTERN + '/forever/:code',
```

Comment:
code is optional right?

Reply:
Good catch! Will not merge this now I realise since we have to change the url i market-web first